tests/core/math/test_math_funcs.h

```diff
@@ -54,6 +54,8 @@ TEST_CASE("[Math] C++ macros") {
 	CHECK(SIGN(-5) == -1.0);
 	CHECK(SIGN(0) == 0.0);
 	CHECK(SIGN(5) == 1.0);
+	// check that SIGN(NAN) returns NAN.
+	CHECK(SIGN(NAN) != SIGN(NAN));
```

Comment:
```suggestion
	CHECK(Math::is_nan(SIGN(NAN)));
```
This doesn't check anything, as any return value will pass this check, you need to use explicit checking

Reply:
Fixed.